From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 5 Mar 2018 21:11:39 +0100
Subject: [PATCH] {BUG in 4.16-rc3} certs/blacklist.h: fix section
 inconsistency

clang reports:

    certs/blacklist_nohashes.c:4:12: error: section does not match
    previous declaration [-Werror,-Wsection]
    const char __initconst *const blacklist_hashes[] = {
               ^
    include/linux/init.h:52:21: note: expanded from macro '__initconst'
    #define __initconst     __section(.init.rodata)
                            ^
    include/linux/compiler_types.h:245:39: note: expanded from macro '__section'
    # define __section(S) __attribute__ ((__section__(#S)))
                                          ^
    certs/blacklist.h:3:19: note: previous attribute is here
    extern const char __initdata *const blacklist_hashes[];
                      ^
    include/linux/init.h:51:20: note: expanded from macro '__initdata'
    #define __initdata      __section(.init.data)
                            ^
    include/linux/compiler_types.h:245:39: note: expanded from macro '__section'
    # define __section(S) __attribute__ ((__section__(#S)))
                                          ^
---
 certs/blacklist.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/certs/blacklist.h b/certs/blacklist.h
index 150d82da8e99..1efd6fa0dc60 100644
--- a/certs/blacklist.h
+++ b/certs/blacklist.h
@@ -1,3 +1,3 @@
 #include <linux/kernel.h>
 
-extern const char __initdata *const blacklist_hashes[];
+extern const char __initconst *const blacklist_hashes[];
-- 
